internal/http/headers.go

```diff
@@ -91,6 +91,14 @@ const (
 	AmzBucketReplicationStatus    = "X-Amz-Replication-Status"
 	AmzSnowballExtract            = "X-Amz-Meta-Snowball-Auto-Extract"
 
+	// MinIOSnowballIgnoreDirs will skip creating empty directory objects.
+	MinIOSnowballIgnoreDirs = "X-Amz-Meta-Minio-Snowball-Ignore-Dirs"
+	// MinIOSnowballIgnoreErrors will ignore recoverable errors, typically single files failing to upload.
+	// An error will be printed to console instead.
+	MinIOSnowballIgnoreErrors = "X-Amz-Meta-Minio-Snowball-Ignore-Errors"
+	// MinIOSnowballPrefix will apply this prefix (plus / at end) to all extracted objects
+	MinIOSnowballPrefix = "X-Amz-Meta-Minio-Snowball-Prefix"
+
```

Comment:
These become reserved x-amz-meta values @klauspost basically they can't used to be set on the object itself as they have prior meaning at this point.

Reply:
@harshavardhana In that respect it isn't really different from `X-Amz-Meta-Snowball-Auto-Extract`. 

Either way, that is why I added "Minio", plus they require `X-Amz-Meta-Snowball-Auto-Extract` to be set to even be active.

Otherwise a larger mc/SDK change is needed to support custom headers.